addons/project/views/project_menus.xml

```diff
@@ -21,19 +21,25 @@
             sequence="1"
         />
         <menuitem
-            name="My Tasks"
-            id="menu_project_management"
-            action="action_server_view_my_task"
-            groups="group_project_user"
+            name="Tasks" 
+            id="menu_project_management" 
             sequence="2"
-        />
-        <menuitem
-            name="All Tasks"
-            id="menu_project_management_all_tasks"
-            action="action_view_all_task"
-            groups="group_project_user"
-            sequence="2"
-        />
+        >
+            <menuitem
+                name="My Tasks"
+                id="menu_project_management_my_tasks"
+                action="action_server_view_my_task"
+                groups="group_project_user"
```

Comment:
The group is not needed since the Project app is only accessible to user with at least Project user group. 🙂